azext_edge/edge/providers/check/base/resource.py

```diff
@@ -468,3 +470,19 @@ def process_custom_resource_status(
                     namespace=namespace,
                     display=Padding(status_text, (0, 0, 0, padding + 4)),
                 )
+
+
+def validate_ref(name: str, namespace: str, ref_type: ValidationResourceType) -> Tuple[str, bool]:
```

Comment:
is this helper going to be specific to mq only?

Reply:
feel like this might be useful across service as we add more runtime resource checks, if I recall correctly dataflow also going to check the runtime resource ref in near terms